www/src/pages/components/modal.js

```diff
@@ -10,6 +10,8 @@ import Modal from '../../examples/Modal';
 import ModalContained from '../../examples/ModalContained';
 import ModalDefaultSizing from '../../examples/ModalDefaultSizing';
 import ModalCustomSizing from '../../examples/ModalCustomSizing';
+import ModalVerticallyCentered from '../../examples/ModalVerticallyCentered';
+import ModalGrid from '../../examples/ModalGrid';
```

Comment:
Lets move all the example files under `examples/Modal/` (and drop the Modal prefix on the file name), like we've been doing with other components

Reply:
Done